core/src/main/java/hudson/model/ParametersAction.java

```diff
@@ -138,10 +138,11 @@ public void createBuildWrappers(AbstractBuild<?,?> build, Collection<? super Bui
         }
     }
 
-    public void buildEnvVars(AbstractBuild<?,?> build, EnvVars env) {
+    @Override
+    public void buildEnvVars(Run<?,?> run, EnvVars env, Node node) {
```

Comment:
This is going to cause [this code](https://github.com/jenkinsci/workflow-job-plugin/blob/e4586916aef819abe4295dbf26dc16b7564ffa56/src/main/java/org/jenkinsci/plugins/workflow/job/WorkflowRun.java#L434-L440) to process Pipeline build parameters twice, until `workflow-job` is updated to disable its own copy of the logic when running on a sufficiently new core version. _Probably_ harmless but needs to be investigated.